__tests__/integration.spec.js

```diff
@@ -85,6 +86,52 @@ describe('toMatchImageSnapshot', () => {
     });
   });
 
+  describe('updates', () => {
```

Comment:
Can we add tests for failing tests? To make sure that failing tests are always updated when ran with `-u` flag no matter what `updatePassedSnapshots` is set to?

Reply:
Done.